### DISCUSSION TITLE
> The queue storage version used is displayed on the individual queue page in the management UI

### DISCUSSION THREAD

**Discussion by 0Ams** [rabbitmq/rabbitmq-server#10170] [Other] [answered]
> The queue storage version used is displayed on the individual queue page in the management UI

_Originally posted by @michaelklishin in https://github.com/rabbitmq/rabbitmq-server/discussions/10137#discussioncomment-7890645_

What version do you start seeing?
I can't find version view on rabbitmq@3.12.10. 
Can you show me an example of how classic version looks?

**Comment by michaelklishin** [comment, marked as answer]
Management UI => Queues => Individual queue => a field named `Queue storage version` or something like that.

**Comment by michaelklishin** [comment]
If matched by policy, the key will also be listed in `Effective policy definition`.

Maybe the key can be hidden in some cases, e.g. when the version is the default. Querying the API endpoint will tell.
